app/dashboard/src/layouts/Settings/settingsData.tsx

```diff
@@ -238,9 +239,43 @@ export const SETTINGS_TAB_DATA: Readonly<Record<SettingsTabType, SettingsTabData
             type: SettingsEntryType.input,
             nameId: 'localRootPathSettingsInput',
             getValue: (context) => context.localBackend?.rootPath ?? '',
-            setValue: (context, value) => context.updateLocalRootPath(value),
+            setValue: async (context, value) => {
+              context.updateLocalRootPath(value)
+              await Promise.resolve()
```

Comment:
btw you can just mark the function as async, no need to `await Promise.resolve()`

Reply:
we can't because of eslint (while technically we could remove the rule, i think it makes sense to leave it out since it's helpful in the vast majority of cases